stories/Colors/Colors.jsx

```diff
@@ -1,114 +1,30 @@
 import React from 'react';
 import propTypes from 'prop-types';
-import { space, color, fontSize } from 'styled-system';
-import chroma from 'chroma-js';
-import styled from '../../src/kfFeels/kfReactEmotion';
-import { H3 } from '../../src/components/Typography';
-import { Paragraph } from '../../src/components/Typography/Text';
-import { Box } from '../../src/components/Layout';
-import { wcagGrade } from '../../src/utils/colors';
 
 /**
  * A basic square swatch to display a color
  */
-/* eslint-disable no-shadow */
```

Comment:
I think we can keep the storybook specific components but just style them with tailwind

Reply:
Need to re-write them cause we no longer have emotion.
Also not sure how to pull out hex values.